opm/common/OpmLog/LogBackend.cpp

```diff
@@ -31,25 +31,51 @@ namespace Opm {
     {
     }
 
-    void LogBackend::configureDecoration(std::shared_ptr<MessageFormatterInterface> formatter)
+    void LogBackend::setMessageFormatter(std::shared_ptr<MessageFormatterInterface> formatter)
     {
-        formatter_ = formatter;
+        m_formatter = formatter;
+    }
+
+    void LogBackend::setMessageLimiter(std::shared_ptr<MessageLimiter> limiter)
+    {
+        m_limiter = limiter;
+    }
+
+    void LogBackend::addMessage(int64_t messageFlag, const std::string& message)
+    {
+        // Forward the call to the tagged version.
+        addTaggedMessage(messageFlag, "", message);
     }
 
     int64_t LogBackend::getMask() const
     {
         return m_mask;
     }
 
-    bool LogBackend::includeMessage(int64_t messageFlag)
+    bool LogBackend::includeMessage(int64_t messageFlag, const std::string& messageTag)
     {
-        return ((messageFlag & m_mask) == messageFlag) && (messageFlag > 0);
+        // Check mask.
+        const bool included = ((messageFlag & m_mask) == messageFlag) && (messageFlag > 0);
+        if (!included) {
+            return false;
+        }
+
+        // Use the message limiter (if any).
+        MessageLimiter::Response res = m_limiter
+            ? m_limiter->handleMessageTag(messageTag)
+            : MessageLimiter::Response::PrintMessage;
+        if (res == MessageLimiter::Response::JustOverLimit) {
+            // Special case: add a message to this backend about limit being reached.
+            std::string msg = "Message limit reached for message tag: " + messageTag;
+            addTaggedMessage(messageFlag, "", msg);
```

Comment:
>If we could weaken the message to say that there may be a log file, perhaps called something.PRT we would not be saying things that could be outright false, would that be good enough?

Yes, that would be enough,  the purpose if just remind the user to check output files. In relperm diagnostics that file called ``.SATFUNCLOG``